### ISSUE TITLE
2.0.0-m4 is missing okhttp-spring-boot-autoconfigure-2.0.0-m4.jar

### ISSUE THREAD

**Issue by wesleysee** [freefair/okhttp-spring-boot#10]
I can't use `2.0.0-m4` as `okhttp-spring-boot-autoconfigure-2.0.0-m4.jar` doesn't seem to exist:

https://jitpack.io/com/github/freefair/okhttp-spring-boot/okhttp-spring-boot-autoconfigure/2.0.0-m4/okhttp-spring-boot-autoconfigure-2.0.0-m4.jar

trying to build and install locally, it looks like `:autoconfigure:install` is being run before `:autoconfigure:jar`, which I think is causing this issue.

```
:autoconfigure:clean
:starters:okhttp:clean
:starters:okhttp2:clean
:starters:okhttp3:clean
:autoconfigure:processResources
:autoconfigure:compileJava
:autoconfigure:classes
:autoconfigure:configureJavadocLinks
:autoconfigure:javadoc
:autoconfigure:javadocJar
:autoconfigure:sourcesJar
:autoconfigure:install
:autoconfigure:jar
:starters:okhttp3:compileJava NO-SOURCE
:starters:okhttp3:processResources NO-SOURCE
:starters:okhttp3:classes UP-TO-DATE
:starters:okhttp3:jar
:starters:okhttp:compileJava NO-SOURCE
:starters:okhttp:processResources NO-SOURCE
:starters:okhttp:classes UP-TO-DATE
:starters:okhttp:jar
:starters:okhttp3:javadoc NO-SOURCE
:starters:okhttp:javadoc NO-SOURCE
:starters:okhttp:javadocJar
:starters:okhttp:sourcesJar
:starters:okhttp:install
:starters:okhttp2:compileJava NO-SOURCE
:starters:okhttp2:processResources NO-SOURCE
:starters:okhttp2:classes UP-TO-DATE
:starters:okhttp2:jar
:starters:okhttp2:javadoc NO-SOURCE
:starters:okhttp2:javadocJar
:starters:okhttp2:sourcesJar
:starters:okhttp2:install
:starters:okhttp3:javadocJar
:starters:okhttp3:sourcesJar
:starters:okhttp3:install
```

**Comment by larsgrefer** [comment]
I will have a look at it

**Comment by sunhacker** [comment]
What's the reason?

**Comment by larsgrefer** [comment]
This should be fixed with 2.0.0-rc2
